cli/pcluster/commands.py

```diff
@@ -144,22 +147,22 @@ def create(args):  # noqa: C901 FIXME!!!
     except ClientError as e:
         LOGGER.critical(e.response.get("Error").get("Message"))
         sys.stdout.flush()
-        if batch_temporary_bucket:
-            utils.delete_s3_bucket(bucket_name=batch_temporary_bucket)
+        if batch_bucket_created:
```

Comment:
Look like a bit of code duplication. Could this block be be moved into a finally block?

Reply:
It's a good idea, unfortunately the `KeyboardInterrupt` has a different behaviour, so we cannot put the code in common.